sdk-dotnet/LittleHorse.Sdk/Utils/FileManager.cs

```diff
@@ -0,0 +1,27 @@
+namespace LittleHorse.Sdk.Utils
+{
+    public class FileManager
+    {
+        internal static Dictionary<string, string> GetDictFromPath(string filePath)
+        {
+            try
+            {
+                Dictionary<string, string> properties = new Dictionary<string, string>();
+                foreach (var tuples in File.ReadLines(filePath)
+                             .Select(line => line.Trim().Split('='))
```

Comment:
I think this only cover the happy path. I think we should ignore lines that are not in K=V format

Reply:
What is the extra line?, This code first is selecting lines that contains = then subselecting key and values without spaces and finally grouping all by keys

Reply:
I would do:

a class Properties, with a private method parseEntry(string), it receives a string and it split it or ignore it in case it starts with a special character (ex #), it returns a pair entry, it throws an exception if the format is not right. then i would have a put(key, value)  (public), it trims and sets the value into a dict, also a load(path) method it reads all the lines, inside the loop it calls parseEntry, and then it calls the method put.

Reply:
Is there no Standard Library implementation of parsing a properties file? Go, Java, and Python all have that in the standard library so you don't need to do it yourself, nor do you need a real third-party dependency.

Reply:
@coltmcnealy-lh No, dotnet has one for .xml, .json and .ini files :(( https://learn.microsoft.com/en-us/dotnet/core/extensions/configuration-providers

Reply:
I think littlehorse.config uses a `ini` format.

Reply:
yes, it worked with the ini configuration :D